docs/Observability/2. prometheus.md

```diff
@@ -0,0 +1,20 @@
+# Prometheus method
```

Comment:
In this document, we describe all types of data. We need to assist the user with both metrics and logging configuration. Therefore, let's come up with section names for these two approaches and add details about log collection. Alternatively, we could add another document about logs.

Reply:
Done. Created the dedicated "topic"